src/grandorgue/dialogs/GOOrganDialog.cpp

```diff
@@ -840,12 +828,16 @@ void GOOrganDialog::FillTree(wxTreeItemId parent, GOPipeConfigNode &config) {
     e = m_Tree->AppendItem(parent, config.GetName(), -1, -1, data);
   for (unsigned i = 0; i < config.GetChildCount(); i++)
     FillTree(e, *config.GetChild(i));
-  m_Tree->Expand(e);
+  // m_Tree->Expand(e);
```

Comment:
Is the commented code expected to be used?

Reply:
No. It was before, but I removed it.